rest/src/main/java/fi/otavanopisto/pyramus/rest/StudentRESTService.java

```diff
@@ -1746,9 +1753,11 @@ public Response createStudentContactLogEntry(@PathParam("ID") Long id, fi.otavan
       return Response.status(Status.FORBIDDEN).build();
     }
     
+    User loggedUser = sessionController.getUser();
```

Comment:
All REST endpoints related to contact log entries and their comments should be annotated with...

`@RESTPermit(handling = Handling.INLINE)`

...rather than permissions in `StudentContactLogEntryPermissions` (which then becomes obsolete)